server/controllers/mainController.js

```diff
@@ -1,3 +1,29 @@
-exports.novelsPage = (req, res) => {
-  res.render("../views/main");
+const mongoose = require('mongoose');
+const Novel = mongoose.model('Novel');
+const path = require('path');
+const fs = require('fs');
+const util = require('util');
+const year = require('year');
+
+exports.mainPage = (req, res) => {
+  const copyrightYear = year();
+  res.render("../views/index", {copyrightYear});
+}
+
+
+exports.novelsPage = async (req, res) => {
+  try{const bookChapterNr =  req.params.chapterNr * 1;
```

Comment:
Using `parseInt()` could make this clearer

Reply:
Updated the method used to parce String into Int
![parseInt](https://user-images.githubusercontent.com/42120918/109331392-b2f63d00-7854-11eb-9a59-b5c982820f74.JPG)